catroid/src/main/java/org/catrobat/catroid/utils/StatusBarNotificationManager.java

```diff
@@ -331,4 +334,11 @@ private void closeNotificationBar() {
 			sendBroadcast(it);
 		}
 	}
+
+	public static int getUniqueNotificationId() {
+		if (uniqueNotificationIdCounter == null) {
+			uniqueNotificationIdCounter = new AtomicInteger(0);
+		}
```

Comment:
since it's not a static method anymore with the latest changes, it might be removed again, am i right?